<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CConfigSimulatorComponent</class>
 <widget class="QFrame" name="CConfigSimulatorComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="vl_ConfigSimulator">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_Simulator">
     <property name="title">
      <string>Select your simulator(s)</string>
     </property>
     <layout class="QGridLayout" name="gl_Simulator">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="1" column="3" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="cb_XP">
        <property name="text">
         <string>XPlane (32,64-bit)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="cb_P3D">
        <property name="text">
         <string>P3D (64-bit)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="cb_FSX">
        <property name="text">
         <string>FSX (32-bit)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="cb_FS9">
        <property name="text">
         <string>FS9 (32-bit)</string>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="4" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="cb_FG">
        <property name="text">
         <string>FlightGear (64-bit)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_SimulatorSetup">
     <property name="title">
      <string>Setup of each individual simulator</string>
     </property>
     <layout class="QVBoxLayout" name="vl_GroupSimulator">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_SimulatorHint">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Simulator directories: &lt;span style=&quot; font-weight:600;&quot;&gt;Normally&lt;/span&gt; there is &lt;span style=&quot; font-weight:600;&quot;&gt;no need&lt;/span&gt; to override the defaults. But if the simulator directories / excludes are incorrect you can set individual values here.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="swift::gui::components::CSettingsSimulatorBasicsComponent" name="comp_SettingsSimulator">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="vs_SimulatorComponent">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CSettingsSimulatorBasicsComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/settingssimulatorbasicscomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cb_P3D</tabstop>
  <tabstop>cb_FSX</tabstop>
  <tabstop>cb_FS9</tabstop>
  <tabstop>cb_XP</tabstop>
  <tabstop>cb_FG</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
